superset-frontend/src/common/components/Dropdown.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { Dropdown as AntdDropdown } from 'src/common/components';
+import { css } from '@emotion/core';
+import { styled } from '@superset-ui/core';
+
+const dotStyle = css`
+  width: 3px;
+  height: 3px;
+  border-radius: 1.5px;
+  background-color: #bababa;
+`;
+
+const MenuDots = styled.div`
+  ${dotStyle};
+  font-weight: 400;
+  display: inline-flex;
+
+  &:hover {
+    background-color: #20a7c9;
+
+    &::before,
+    &::after {
+      background-color: #20a7c9;
+    }
+  }
+
+  &::before,
+  &::after {
+    position: absolute;
+    content: ' ';
+    ${dotStyle};
+  }
+
+  &::before {
+    transform: translateY(-5px);
```

Comment:
```suggestion
    transform: translateY(-${({ theme }) => theme.gridUnit}px);
```
Not sure if 4px is close enough for this use case